modules/guides/guides.libraries.yml

```diff
@@ -9,3 +9,11 @@ guide:
   css:
     theme:
       css/guides.guide.css: {}
+
+in_page_navigation:
+  version: 1.0
```

Comment:
Dunno what this line here means, and I'm not sure if it's needed either. If it's not needed, then pls don't add such lines, as they might be confusing.

Reply:
It's the library definition for the in-page-nav.